app/Cart.php

```diff
@@ -0,0 +1,93 @@
+<?php
+// Anything under the "/app" folder should have the namespace "App".
+namespace App;
+
+/**
+ * Magement of the item in the shopping cart
+ */
+class Cart
+{
+    /**
+     * Declare properties
+     *
+     * @var array $items A list of product items
+     */
+    private $items = [];
+
+    /**
+     * @var str $item An item within the cart
+     */
+    private $item;
+
+    /**
+     * @var int $totalItemCount The number of item in the cart
+     */
+    private $totalItemCount;
+
+    /**
+     * @var str $renderHtml The HTML cart listing layout
+     */
+    private $renderHtml = 'foo';
```

Comment:
We now have a method called `renderHtml` AND we have a property called `renderHtml`. Which should be a noun, and which a verb?